src/txmempool.cpp

```diff
@@ -661,26 +661,25 @@ void CTxMemPool::removeConflicts(const CTransaction &tx)
     }
```

Comment:
In "mempool: Avoid expensive loop in removeForBlock during IBD" 41ad2be4340d7b18a4669aa9becef8936597f000

nit: change commit title to  "mempool: avoid looping `blk` `txs` when mempool is empty in `removeForBlock`"